samples/graph-teams-tag/nodejs/client/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Meeting Stage View Sample"
+    />
+    <title>Meetings Stage View</title>
```

Comment:
This is not meeting stage view sample :)

Reply:
Done 😄